<launch>
    <node pkg="long_term_server" type="test.py" name="test_node" required="true"/>
</launch>
